source/declarations.tex

```diff
@@ -4422,6 +4422,272 @@
 \indextext{declaration!default argument|)}%
 \indextext{declarator!meaning of|)}
 
+\rSec3[except.spec]{Exception specifications}%
```

Comment:
I'm a bit uneasy putting this under "meaning of declarators", which focuses on the recursive decomposition.

Right after "function definitions" (two levels up) feels better to me.   We talk an awful lot about a "function declaration having a non-throwing exception declaration" in the text, and a declarator isn't a complete declaration, so this feels like the wrong spot.

Reply:
How about after [dcl.fct.spec] instead?

Reply:
This is the section about _decl-specifier_s. A _noexcept-specifier_ isn't a _decl-specifier_. It's more like an attribute, considering its trailing placement.

Reply:
After carefully hunting through [decl], I think the best place for this right now is immediately following [dcl.fct] that has the grammar for the optional _noexcept-specifier_ on a function (type) declaration.

I will get this branch into a non-conflicting state, and suggest we discuss further in Wrocław.